gulpfile.js

```diff
@@ -137,6 +139,16 @@ gulp.task("watch", function () {
   gulp.watch(dirs.img + "/**/*.*", ["images"]);
 });
 
+gulp.task("war", function () {
```

Comment:
May I suggest to make this task depend on `dist` ? This way we can actually go to the next step and get rid of the `dist/` folder in the repo and be sure it is there when `npm run war` runs.